src/sunburst.js

```diff
@@ -115,7 +114,7 @@ export default Kapsule({
     state.svg = el.append('svg');
     state.canvas = state.svg.append('g')
       .style('font-family', 'sans-serif')
-      .style('font-size', `${TEXT_FONTSIZE}px`);
+      .style('font-size', `${state.textSize}px`);
```

Comment:
This needs to be moved to the `update` section so it can react to font size changes.